src/main/java/de/komoot/photon/elasticsearch/IndexSettings.java

```diff
@@ -72,7 +72,7 @@ public IndexSettings setSynonyms(String synonymFile) throws IOException {
             if (line.indexOf(' ') >= 0) {
                 throw new RuntimeException("Synonym list must not contain any spaces or multi word terms.");
             }
-            synonyms.put(line);
+            synonyms.put(line.toLowerCase());
```

Comment:
`line.toLowerCase(Locale.ROOT)` ?

Reply:
It should be "whatever ES uses", to be precise.